### ISSUE TITLE
Embed newsletter subscription form on homepage

### ISSUE THREAD

**Issue by bloudermilk** [climatescape/climatescape.org#307]
We should embed this on the homepage in place of the "Subscribe Here" button.

```html
<iframe src="https://climatescape.substack.com/embed" width="480" height="320" style="border:1px solid #EEE; background:white;" frameborder="0" scrolling="no"></iframe>
```

Not sure what this actually looks like. Let's throw it in there and check in with #design for feedback.

**Comment by RNR1** [comment]
the most reasonable way felt like swapping the whole section, though there was a meaningful text there, let's have a discussion about how it feels right in the pr i just opened.

![Screen Shot 2020-06-09 at 23 08 37](https://user-images.githubusercontent.com/45559220/84194738-288f6280-aaa6-11ea-9dce-ee294756db52.png)

**Comment by bloudermilk** [comment]
Fixed by #318. Nice work, @RNR1 !
